libs/shared/profile-form/src/lib/shared-profile-form.tsx

```diff
@@ -248,521 +249,501 @@ export const SharedProfileForm = (props: SharedProfileFormProps) => {
+              fullWidth
+            />
+          </Grid>
+
+          <Grid item md={6}>
+            <TextField
+              id="company"
+              name="company"
+              defaultValue={(props.profile && props.profile.company) || ''}
+              type="text"
+              variant="outlined"
+              label="Company"
+              inputRef={register}
+              fullWidth
+            />
+          </Grid>
+
+          <Grid item md={6}>
+            <FormControl
+              variant="outlined"
+              fullWidth
+              error={!!errors.categories}
+            >
+              <InputLabel id="categoriesLabel">Categories</InputLabel>
+              <Controller
+                name="categories"
                 defaultValue={
-                  (props.profile && props.profile.fieldOfProfession) || ''
+                  props.profile && props.profile.categories
+                    ? props.profile.categories
+                    : []
+                }
+                control={control}
+                rules={{ validate: (value) => validateCategories(value) }}
+                onChange={([selected]) => {
+                  return selected;
+                }}
+                as={
+                  <Select
+                    id="categories"
+                    label="Categories"
+                    name="categories"
+                    value={categories}
+                    multiple
+                    onChange={handleCategoriesChange}
+                    input={
+                      <OutlinedInput
+                        id="select-multiple-chip"
+                        label="Categories"
+                      />
+                    }
+                    inputRef={register}
+                    error={!!errors.categories}
+                  >
+                    {props.categories.map((category: Category) => (
+                      <MenuItem key={category._id} value={category.value}>
+                        {category.title}
+                      </MenuItem>
+                    ))}
+                    <MenuItem
+                      dense
+                      divider
+                      value={[]}
+                      onClickCapture={stopImmediatePropagation}
+                      onKeyDown={(event) => event.stopPropagation()}
+                      className="new-category__form"
+                    >
+                      <TextField
+                        id="new-category"
+                        name="new-category"
+                        type="text"
+                        variant="outlined"
+                        size="small"
+                        label="Add new"
+                        className="new-category__input"
+                        value={newCategory.title}
+                        onChange={handleNewCategoryInput}
+                      />
+                    </MenuItem>
+                    {newCategory.title.length > 0 && (
```

Comment:
length con't be lower than 0 so any length above 0 is equal true: !!newCategory.title.length